### DISCUSSION TITLE
When multiple column data are presented in graph.

### DISCUSSION THREAD

**Discussion by Jeriousman** [pyg-team/pytorch_geometric#4137] [Q&A] [answered]
Let's say we have data features like [age, gender, score, income]. When we make node feature for this person in a hetero graph data, ```data['person'].x = ... # [num_persons, num_features_person]``` Do we normally concatenate [age, gender, score, income] for num_features_person? Is it how normally we do? If it is true the shape will be ```data['person'].x = [N, 4] I presume. Or should we make different features as a single feature in graph like ```data['person'].age, data['person'].gender, data['person'].score, data['person'].income? 

It would be much appreciated to know common practice. Thank you.

**Comment by rusty1s** [comment, marked as answer]
`data["person"].x` should include all features of a person. As such, one concatenates all these different features into a single big feature matrix.

**Reply by Jeriousman** [reply]
Thank you for your great help!
